test/esjs.spec.js

```diff
@@ -346,6 +363,33 @@ describe('.search()', () => {
         expect(searchIds(results)).to.eql([4]);
       });
     });
+
+    context('given an exact string matching a "keyword" field', () => {
+      const results = idx.search('PUBLISHED');
+
+      it('returns 0 results', () => {
+        expect(results.length).to.be(0);
+      });
+    });
+
+    context('given a term matching a "keyword" field', () => {
+      const results = idx.search({
+        must: {
```

Comment:
Would this also work as?
```
must: { match: { term: { status: 'NEW' } } }
```

Reply:
Just checked; that is not working properly. Looking into it

Reply:
After looking through the elasticsearch docs, I think that query may be invalid. [`match`](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-match-query.html), and [`term`](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-term-query.html) are [leaf query clauses](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html) and, from what I've gathered, should be positioned adjacently within a compound query, not nested within one another.

e.g.:
```javascript
must: {
  term: { status: 'NEW' }
}
```

^^ above syntax is working as expected

Reply:
Sorry bad example!

I meant: 
```
must: {
  match: 'foo',
  term: { status: 'NEW' },
},
```

I think there should be a test for this, as it's different than filtering as tested below.